### PR TITLE
fix: auto label

### LINKED ISSUE #857: fix auto labelling of pull requests
We have a `autolabler` setting in `release_drafter.yml` which should automatically apply labels to pull requests but it doesn't happen.

Identify the issue of why it doesn't automatically add labels based on the title of the PR or branch name and fix it.

### PR DESCRIPTION
**Pull request by alankritdabral**
**Summary:**  
Earlier only pr which were from the tailcall/tailcall were used add labels on the pr 
But if any pr from the forked branches were not able to add labels to pr

**Issue Reference(s):**  
/fixes #857
/claim #857

**Build & Testing:**

- [ ] I ran `cargo test` successfully.
- [ ] I have run `./lint.sh --mode=fix` to fix all linting issues raised by `./lint.sh --mode=check`.

**Checklist:**

- [ ] I have added relevant unit & integration tests.
- [ ] I have updated the [documentation] accordingly.
- [ ] I have performed a self-review of my code.
- [ ] PR follows the naming convention of `<type>(<optional scope>): <title>`

[documentation]: https://github.com/tailcallhq/tailcall/tree/main/docs

### CODE REVIEW

**Review by tusharmath — APPROVED** [review]

**Comment by tusharmath** [comment]
@alankritdabral Can we create a separate ci file for running release drafter as a cron job every minute instead?

**Comment by alankritdabral** [comment]
> @alankritdabral Can we create a separate ci file for running release drafter as a cron job every minute instead?

Instead of this I have another approach 
in the ci.yml we can stop running  the `draft_release` job on pull request's as it is already running on main 
and now we create a seprate ci file that will run on `pull_request_target` the code for new file would look like 

```
name: run release-drafter

on:
  pull_request_target:
jobs:

  draft_release:
    name: Draft Release
    runs-on: ubuntu-latest
    permissions:
      contents: write
      pull-requests: write
    steps:
      - name: Checkout Current Branch (Fast)
        uses: actions/checkout@v4
    
      - id: create_release
        uses: release-drafter/release-drafter@v5
        if: (github.event_name == 'pull_request') || (github.event_name == 'push' && github.ref == 'refs/heads/main')
        env:
          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
        with:
          config-name: release-drafter.yml
```

File: .github/workflows/release-drafter.yml [hunk 1]
```diff
@@ -0,0 +1,38 @@
+name: Release Drafter
+
+on:
+  schedule:
+    - cron: "*/5 * * * *"
+  pull_request_target:
+    branches:
+      - "**"
+    paths-ignore: ["docs/**", "**.md"]
```
Comment (amitksingh1490) [inline .github/workflows/release-drafter.yml:9]: why is this added

File: .github/workflows/release-drafter.yml [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+
+      - name: Set Output for Later Jobs
+        id: set_output
+        run: |
+          echo "create_release_name=${{ steps.create_release.outputs.name }}" >> $GITHUB_OUTPUT
+          echo "create_release_id=${{ steps.create_release.outputs.id }}" >> $GITHUB_OUTPUT
+    outputs:
+      create_release_name: ${{ steps.set_output.outputs.create_release_name }}
+      create_release_id: ${{ steps.set_output.outputs.create_release_id }}
```
Comment (amitksingh1490) [inline .github/workflows/release-drafter.yml:38]: this output will not get reflected in the tailcall build. it will introduce a bug

**Comment by amitksingh1490** [comment]
This won't work, Closing this, let me work on this as its urgent for us.
